### DISCUSSION TITLE
PNP search results on LocalPeopleResults not returning all People

### DISCUSSION THREAD

**Discussion by Kris3D** [microsoft-search/pnp-modern-search#3615] [Show and tell]
I followed the documentation to the letter to build a peoples search page. But the results part does not return all people present.
Even with only a results webpart and * as search param it only returns about 80 records instead of all 130. What can be wrong?
Using the latest version 4.10.2

**Comment by wobba** [comment]
You will only return people who are stored in the SharePoint User Profile store. Go to https://<tenant>-admin.sharepoint.com/_layouts/15/TenantProfileAdmin/ManageUserProfileServiceApplication.aspx to browse profiles.

**Comment by Kris3D** [comment]
Hi Mikael

All 250 users are present in the SharePoint User Profile store. But they do not all show in the results. 

Met vriendelijke groeten,

Kris De Pril

Op 9 mrt 2024 om 14:32 heeft Mikael Svenson ***@***.***> het volgende geschreven:

SharePoint User Profile store.

**Reply by wobba** [reply]
How are you checking? One large page size? Paging? Comparing with e.g. Classic search UX? Have you checked the crawl log that the profiles you don't get are indexed? Do you get them when querying on e.g UPN but not on wildcard? 

Adding details helps. Either way it's an API issue and not something the web parts can control.

**Reply by Kris3D** [reply]
I checked against https://contoso-admin.sharepoint.com/_layouts/15/TenantProfileAdmin/ManageUserProfileServiceApplication.aspx
Here all profiles show up on search.
On the search result pagin on. Where can I find the crawl log? 
In the search template i just have {searchTerms} of {searchTerms}* or *. All with the same result.
Tried in another tenant with same results. Only about 14 profiles of at least 50.
How do I put in a UPN query?

**Reply by wobba** [reply]
Saerching in UPA is not the same as if the profiles are indexed. See https://pnp.github.io/powershell/cmdlets/Get-PnPSearchCrawlLog.html#example-4 for an example on crawl log and profiles. And try a page size of 200.

**Reply by Kris3D** [reply]
I tried example 3 and 4 and neither returns any results. I ran it against https://contoso.sharepoint.com
That's also the sps where I use the PNP search results. Where do I go from here?

**Reply by wobba** [reply]
If you can repro using classic search at e.g. https://tenant.sharepoint.com/search/Pages/peopleresults.aspx then log a support case with Microsoft. From what you say it sounds more like a data issue, not web part issue.

**Reply by Kris3D** [reply]
Classic search is giving me the same results. Ticket submitted.
